packages/radix-ui-themes/src/components/index.css

```diff
@@ -1,9 +1,13 @@
 @import './reset.css';
-/* Import Skeleton before other components so that its default border-radius doesn’t have a higher specificity */
-@import './skeleton.css';
 @import './animations.css';
 @import './layout.css';
 
+/* Import Skeleton before other components so that its default border-radius doesn’t have a higher specificity */
+@import './skeleton.css';
+
+/* Import Text before other components as it’s commonly extended */
+@import './text.css';
```

Comment:
Not sure how I feel about this as a solution, what is the actual root cause?

Reply:
Not sure how it had worked before to be honest, this seems like an obviously problematic area for components that extend text and target the same selectors like `[data-accent-color]`